packages/pg-connection-string/index.js

```diff
@@ -69,16 +68,23 @@ function parse(str) {
     config.ssl = {}
   }
 
-  if (config.sslcert) {
-    config.ssl.cert = fs.readFileSync(config.sslcert).toString()
-  }
+  try {
+    if (config.sslcert) {
+      const fs = require('fs')
+      config.ssl.cert = fs.readFileSync(config.sslcert).toString()
+    }
 
-  if (config.sslkey) {
-    config.ssl.key = fs.readFileSync(config.sslkey).toString()
-  }
+    if (config.sslkey) {
+      const fs = require('fs')
+      config.ssl.key = fs.readFileSync(config.sslkey).toString()
+    }
 
-  if (config.sslrootcert) {
-    config.ssl.ca = fs.readFileSync(config.sslrootcert).toString()
+    if (config.sslrootcert) {
+      const fs = require('fs')
+      config.ssl.ca = fs.readFileSync(config.sslrootcert).toString()
+    }
+  } catch (e) {
+    throw new Error('filesystem not supported')
```

Comment:
Or any of the files wasn’t readable. Just the `require` should go in `try`, ideally.

Reply:
Yep! Fixing.